src/components/molecules/UiLoader/UiLoader.vue

```diff
@@ -81,12 +67,18 @@ const props = defineProps({
     default: 'spinner',
   },
   /**
-   * Use this props to pass attributes to internal child components
+   * Use this props to set transition type.
    */
-  loaderAttrs: {
-    type: Object as PropsAttrs,
-    default: () => ({
-    }),
+  transitionType: {
+    type: String as PropType<LoaderTransitionType>,
+    default: 'if',
+  },
+  /**
+   * Use this props to set transition name.
+   */
+  name: {
```

Comment:
@ArkadiuszKrzymienski I add a name for the transition name to set the name of the transition. Mostly it will be `fade` but the user may want to use other transition in some cases.